API.md

```diff
@@ -1,12 +1,120 @@
 # Authmosphere API Documentation
 
+## Token Cache
+
+Class to request and cache tokens on client-side.
+
+#### Usage
+
+```typescript
```

Comment:
In other examples we added `import` statement (though not everywhere). Shall we be consistent?